src/lib/preferences.ts

```diff
@@ -93,7 +93,8 @@ const defaults = {
 
     "supporter.currency": "auto",
     "supporter.interval": "month",
-    "tournaments-tab": "correspondence",
+    "tournaments-tab": "schedule",
```

Comment:
My initial reaction on having this be the default was negative because there's no actionable things on the page allowing players to join anything, but maybe that's not a bad thing as hopefully the next thing someone interested in joining will do is find the live/correspondence sub-tabs, and in doing that they'll learn that there are two tabs there depending on the type of tournament you're interested in joining. So, lets try it :)

Reply:
That's a valid concern.  As a mentioned in the commit message, this was mostly prompted by a suggestion in the bug.

So anyway if this part needs to be rolled back that's fine by me :)